### PR TITLE
[Consensus Voting] Fix all integration tests

### PR DESCRIPTION
**Pull request by zhangchiqing**
Fix and enable all integration tests. 

They are all passing on local

### CODE REVIEW

File: engine/consensus/ingestion/core.go [hunk 1]
```diff
@@ -57,6 +57,8 @@ func NewCore(
 // All other errors are unexpected and potential symptoms of internal state corruption.
 func (e *Core) OnGuarantee(originID flow.Identifier, guarantee *flow.CollectionGuarantee) error {
 
+	e.log.Info().Msgf("on guarantee called for guarantee %v from %v", guarantee.ID(), originID)
```
Comment (jordanschalm) [inline engine/consensus/ingestion/core.go:60]:
```suggestion
	e.log.Debug().Msgf("on guarantee called for guarantee %v from %v", guarantee.ID(), originID)
```
Not sure if we want info-level for this common of an event

File: integration/testnet/network.go [hunk 2]
```diff
@@ -547,33 +549,45 @@ func PrepareFlowNetwork(t *testing.T, networkConf NetworkConfig) *FlowNetwork {
 			accessNodeIDS = append(accessNodeIDS, n.NodeID.String())
 		}
 	}
-	require.True(t, len(accessNodeIDS) >= DefaultMinimumNumOfAccessNodeIDS, fmt.Sprintf("at-least %d access node that is not a ghost must be configured for test suite", DefaultMinimumNumOfAccessNodeIDS))
+	require.True(t, len(accessNodeIDS) >= DefaultMinimumNumOfAccessNodeIDS,
+		fmt.Sprintf("at-least %d access node that is not a ghost must be configured for test suite", DefaultMinimumNumOfAccessNodeIDS))
```
Comment (jordanschalm) [inline integration/testnet/network.go:553]:
```suggestion
	require.GreaterOrEqualf(t, len(accessNodeIDS), DefaultMinimumNumOfAccessNodeIDS,
		"at-least %d access node that is not a ghost must be configured for test suite", DefaultMinimumNumOfAccessNodeIDS)
```

File: integration/testnet/network.go [hunk 3]
```diff
@@ -547,33 +549,45 @@ func PrepareFlowNetwork(t *testing.T, networkConf NetworkConfig) *FlowNetwork {
...
+		var nodeType = "real"
+		if nodeConf.Ghost {
+			nodeType = "ghost"
+		}
+
+		t.Logf("add docker container type: %s, %v, node id: %v, address: %v", nodeType, nodeConf.ContainerName, nodeConf.NodeID, nodeConf.Address)
 		err = flowNetwork.AddNode(t, bootstrapDir, nodeConf)
 		require.NoError(t, err)
 
+		// ghost nodes will not need any fags
```
Comment (jordanschalm) [inline integration/testnet/network.go:565]:
```suggestion
		// ghost nodes will not need any flags
```

File: integration/tests/consensus/inclusion_test.go [hunk 4]
```diff
@@ -102,34 +102,117 @@ func (is *InclusionSuite) TearDownTest() {
...
+
+		return conID
 	}
+	is.T().Fatalf("timeout sendng collection %x to consensus", colID)
+	return flow.ZeroID
+}
 
+func (is *InclusionSuite) waitUntilCollectionIncludeInProposal(deadline time.Time, sentinel *flow.CollectionGuarantee) *messages.BlockProposal {
+	colID := sentinel.CollectionID
+	// we try to find a block with the guarantee included and three confirmations
```
Comment (jordanschalm) [inline integration/tests/consensus/inclusion_test.go:175]:
```suggestion
	// we try to find a block with the guarantee included
```
We are checking confirmations/finalization later

**Review by jordanschalm — APPROVED** [review]
The changes look good, but I'm not sure all the tests are reliable enough still to enable on CI.

**Review by kc1116 — APPROVED** [review]

**Comment by zhangchiqing** [comment]
bors merge

**Comment by zhangchiqing** [comment]
bors merge

**Comment by zhangchiqing** [comment]
bors merge

**Pull request merged**